docs/Developers/Writing_Native_Bears.rst

```diff
@@ -598,4 +598,10 @@ For example, let's make an aspect bear named SpellingCheckBear.
                         aspect=DictionarySpelling('py'),
                     )
 
+.. note::
+
+    coala uses 1-based line & column convention, i.e., the first line
+    and first column are 1. However, some linters use 0-based convention.
```

Comment:
"linters" is not relevant for native bears where the `Result` objects are created in coala, and the `Result` constructor will raise an exception if '0' are given (and due to our 100% test coverage policy, all `Result` constructors will be tested).